### ISSUE TITLE
EasyAI: No China "Place Units" phase on World War II v3 1941 map (AA50).

### ISSUE THREAD

**Issue by gg405** [triplea-game/triplea#3350]
### Engine version
1.9.0.0.9687

### My Operating System
Windows 10

### Map name
World War II v3 1941

### Can you describe how to trigger the error? (eg: what sequence of actions will recreate it?)
Playing game with all AI except Human British. On Turn 1, Chinese earned 2 infantry, but there was no China placement phase and infantry never made it to the map. Pretty much makes this map/scenario unplayable.

(PS, maybe stop suggesting zipping files, as the zipfile (50 KB) is larger than source file (49 KB). Or perhaps, only in this simple instance.)

[No_china_placement_phase.zip](https://github.com/triplea-game/triplea/files/1885818/No_china_placement_phase.zip)

**Comment by panther2** [comment]
Indeed, I did a quick test to verify that using HardAI a Chinese Placement Phase takes place while usinf EasyAI this phase is skipped.

[savedGames.zip](https://github.com/triplea-game/triplea/files/2404218/savedGames.zip)
